pkg/neorpc/filters.go

```diff
@@ -48,6 +51,43 @@ type (
 		Type   *mempoolevent.Type `json:"type,omitempty"`
 	}
 )
+type ValidFilter interface {
+	IsValid() error
+}
+
+// IsValid checks whether the filter is valid.
+func (f *BlockFilter) IsValid() error {
+	return nil
+}
+
+// IsValid checks whether the filter is valid.
+func (f *TxFilter) IsValid() error {
+	return nil
+}
+
+// IsValid checks whether the filter is valid.
+func (f *NotificationFilter) IsValid() error {
+	if f.Name != nil && runtime.MaxEventNameLen < len(*f.Name) {
+		return errors.New("bad name parameter")
```

Comment:
return fmt.Errorf("NotificationFilter name parameter must be less than %d", runtime.MaxEventNameLen)